src/utils/resultsTable.js

```diff
@@ -19,24 +19,23 @@ such restriction.
 */
 import { capitalize } from 'lodash'
 
-export const resultsTable = array => {
-  let headers = (array.iterationStats ? array.iterationStats[0] ?? [] : []).map(
-    item => {
-      return capitalize(String(item).replace(/^.+\./, ''))
-    }
-  )
-  const changedHeaders = [headers[1], headers[0]]
-  headers = changedHeaders.concat(headers.slice(2))
-  const tableContent = (array.iterationStats ?? [])
-    .slice(1)
-    .map(contentItem => {
-      return [contentItem[1], contentItem[0]]
-        .concat(contentItem.slice(2))
-        .map(item => item ?? '')
+export const resultsTableHeaders = array => {
+  const [headers] = (array.iterationStats ?? []).slice(0, 1).map(item => {
+    return [item[1], item[0]].concat(item.slice(2)).map(header => {
+      const clearHeaderPrefix = String(header).replace(/^.+\./, '').toLocaleLowerCase()
+
+      return {
+        header: capitalize(clearHeaderPrefix),
```

Comment:
we need to rename `header` to `headerId`. Probably it will require to make changes to all tables